Dockerfile

```diff
@@ -1,5 +1,6 @@
 ARG PARENT_VERSION=1.2.5-node14.16.1
 ARG PORT_DEBUG=9229
+ARG PORT=3008
```

Comment:
Need to add an ARG and EXPOSE within the Development definition in this file or this value isn't used for anything.

eg you should end up with something like the below from line 10.

```
ARG PORT
ARG PORT_DEBUG
ENV PORT ${PORT}
EXPOSE ${PORT} ${PORT_DEBUG}
```